src/network/networkd-network-gperf.gperf

```diff
@@ -248,6 +248,7 @@ DHCPv4.RouteMetric,                          config_parse_dhcp_route_metric,
 DHCPv4.RouteTable,                           config_parse_dhcp_or_ra_route_table,                      AF_INET,                       0
 DHCPv4.UseTimezone,                          config_parse_bool,                                        0,                             offsetof(Network, dhcp_use_timezone)
 DHCPv4.ListenPort,                           config_parse_uint16,                                      0,                             offsetof(Network, dhcp_client_port)
+DHCPv4.Port,                                 config_parse_uint16,                                      0,                             offsetof(Network, dhcp_port)
```

Comment:
Please use more specific name, e.g. ServerPort=, TransmitPort=, SendPort= or so.

Reply:
Renamed to `DHCPv4.ServerPort`